src/components/Reports/FinancialAccountsReport/AccountTransactions/AccountTransactions.test.tsx

```diff
@@ -295,9 +300,18 @@ describe('Financial Account Transactions', () => {
 
     it('should export CSV', async () => {
       const mockBlobUrl = 'blob:';
-      buildURI.mockReturnValue(mockBlobUrl);
+      (buildURI as jest.Mock).mockReturnValue(mockBlobUrl);
 
-      const { getByRole } = render(<Components />);
+      const { getByRole } = render(
```

Comment:
You changed this to pass in filters. Is that necessary? Can we revert that change?

Reply:
Oh no, I just am asking if you can remove the `filters` prop from this one test. We weren't passing in custom filters before.